templates/operations.html

```diff
@@ -816,4 +820,13 @@ <h1 id="time-tactic" class="member-title"></h1>
         $('#resultCmd').html(atob(command));
     }
     //# sourceURL=operations.js
+
+    function advHasRepeatableAbilities(selectedAdversary) {
+        if ($('#queuePlanner').val() == 'atomic' && $('#qflow-' + selectedAdversary).data('has-repeatable') != null) {
```

Comment:
let's look at approaches to this without the need to hard code a planner name. Other options may be solving the upstream issue or specifying capabilities of planners. (I.e. this planner can support X and here X would be repeatable abilities)

Reply:
somewhat addressed in https://github.com/mitre/caldera/pull/2019/commits/36fa1d5323d03f1c4a287dc63da691fc12670ebf by adding an `allow_repeatable_abilities` property to planners. Hard coding the planner names is still used for "defaulting" to specific planners in the GUI.